src/web/views/FlowBuilder/sidePanel/properties/ActionModalForm.jsx

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 import { Modal, Button, Radio, OverlayTrigger, Tooltip, Panel, Well } from 'react-bootstrap'
 import Select from 'react-select'
 import axios from 'axios'
+import _ from 'lodash'
```

Comment:
Can't see where it's used in the changed lines

Reply:
That's because it's used within unchanged lines. )

Reply:
Yeah guessed that :/ Did ESLint help catching this error?

Reply:
Yep. )